doc/release-notes/5.10-release-notes.md

```diff
@@ -0,0 +1,340 @@
+
+### Downloading of Guestbook Responses with Fewer Clicks
+
+A download button has been added to the page that lists guestbooks. This saves a click but you can still download responses from the "View Responses" page, as before.
+
+Also, links to the guides about guestbooks have been added in additional places.
+
+### Dynamically Request Arbitrary Metadata Fields from Search API 
+
+The Search API now allows arbitrary metadata fields to be requested when displaying results from datasets. You can request all fields from metadata blocks or pick and choose certain fields.
+
+The new parameter is called `metadata_fields` and the Search API documentation contains details and examples: <https://guides.dataverse.org/en/5.10/api/search.html>
+
+### Solr 8 Upgrade
+
+The Dataverse Software now runs on Solr 8.11.1, the latest available stable release in the Solr 8.x series.
+
+### PostgreSQL Upgrade
+
+A PostgreSQL upgrade is not required for this release but is planned for the next release. See below for details.
+
+## Major Use Cases and Infrastructure Enhancements
+
+Changes and fixes in this release include:
+
+- When creating/updating datasets, users can select from a set of standard licenses configured by the administrator or provide custom terms (if the installation is configured to allow them). (Issue #7440, PR #7920)
+- Users can get better feedback on tabular ingest errors and more information about size limits when uploading files. (Issue #8205, PR #8271)
+- Users can more easily download guestbook responses and learn how guestbooks work. (Issue #8244, PR #8402)
+- Search API users can specify additional metadata fields to be returned in the search results. (Issue #7863, PR #7942)
+- The "Preview" tab on the file page can now show restricted files. (Issue #8258, PR #8265)
+- Users wanting to upload files from GitHub to Dataverse can learn about a new GitHub Action called "Dataverse Uploader". (PR #8416)
+- Users requesting access to files now get feedback that it was successful. (Issue #7469, PR #8341)
+- Users may notice various accessibility improvements. (Issue #8321, PR #8322)
+- Guestbooks now support multi-line text area fields. (Issue #8288, PR #8291)
+- Guestbooks can better handle commas in responses. (Issue #8193, PR #8343)
+- Dataset editors can now deselect a guestbook. (Issue #2257, PR #8403)
+- Administrators with a large `actionlogrecord` table can read docs on archiving and then trimming it. (Issue #5916, PR #8292)
+- Administrators can list locks across all datasets. (PR #8445)
+- Administrators can run a version of Solr that doesn't include a version of log4j2 with serious known vulnerabilities. We trust that you have patched the version of Solr you are running now following the instructions that were sent out. An upgrade to the latest version is recommended for extra peace of mind. (PR #8415)
```

Comment:
I'm not sure what normal protocol is for ordering issues, but I'd put them in order of significance since people read from the beginning and maybe don't go thru the full list, so getting them aware of the most important issues by putting them at the top of the list probably can be helpful.  It seems that log4j fixes and multi-license are the most significant.

Reply:
@lenwiz I struggled a lot with how to talk about log4j. Perhaps we should have a separate section about security updates? There is an order for these bullets but it may not be the most obvious. Users are first, then curators, then administrators, then sysadmins. Basically, I'm trying to put the stuff at the top that affects the most people. Sysadmins, of which there are few, are probably the only ones who know about log4j. So I put it last with the sysadmin stuff.

@landreev I see you gave this comment a 👍 so please feel free to join in. 😄 I'm in no rush to click "resolve conversation".

Bottom line, I'm inclined to leave it alone but I'm open to reordering, even if it goes against by "most people" logic above. 😄